app/views/healthcare/edit.html.slim

```diff
@@ -2,7 +2,7 @@
   - breadcrumbs_for_page root: true do
     - breadcrumb escort_breadcrumb(escort), escort_path(escort)
     - breadcrumb 'Healthcare summary', escort_healthcare_path(escort)
-    - breadcrumb t("healthcare.form.title.#{form.name}")
+    - breadcrumb location_text(t("healthcare.form.title.#{form.name}"), form.location)
```

Comment:
You can use escort.location here, instead of defining another method on the form